content/post/Android_Root101.en.md

```diff
@@ -1,31 +1,32 @@
 +++
 title = "A Guide to Flashing Custom ROMs [Android]"
-slug = "Flashing Custom ROMs 101"
+slug = "custom-rom-101"
 date = "2018-02-25T00:00:00"
+image = "img/rohanvtk/csrom.jpg"
```

Comment:
Since we are having this change in many place, 
Can you try and put a URL of this image rather than the image itself. Would help a ton to save bits of space. Thanks,

Reply:
I reduced the size to 78KB. All my images are now under 90KB. 
💛 #optimized 😄

Reply:
:hatching_chick: